src/puppetlabs/trapperkeeper/authorization/rules.clj

```diff
@@ -149,10 +149,10 @@
   (let [from (requestor request)
         path (:uri request)
         method (:request-method request)
-        authentic? (true? (ring/authorized-authentic? request))]
+        authenticated? (true? (ring/authorized-authentic? request))]
```

Comment:
I was thinking we might as well change the `authorized-authentic?` ring helpers to `authorized-authenticated`.  Sound okay?